logstash-core/src/main/java/org/logstash/ackedqueue/Page.java

```diff
@@ -2,27 +2,28 @@
 
 import java.io.Closeable;
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.BitSet;
-import java.util.List;
+
+import org.logstash.ackedqueue.io.CheckpointIO;
 import org.logstash.ackedqueue.io.LongVector;
 import org.logstash.ackedqueue.io.PageIO;
 
-public abstract class Page implements Closeable {
+public class Page implements Closeable {
```

Comment:
@colinsurprenant I guess we can even make this one `final` now right?

Reply:
+1